src/PartialResponse.Core/Tokenizer.cs

```diff
@@ -51,28 +52,31 @@ public Token NextToken()
                 return new Token(null, TokenType.Eof, this.position);
             }
 
-            Token token;
-
-            TokenType tokenType;
+            var c = this.GetCurrentCharacter();
 
-            if (this.tokens.TryGetValue(this.GetCurrentCharacter(), out tokenType))
+            if (this.tokensMap.TryGetValue(c, out var tokenType))
             {
                 this.TakeCharacter();
 
-                token = new Token(this.buffer.ToString(), tokenType, this.position);
+                return this.CreateToken(tokenType);
             }
-            else if (this.IsWhiteSpace(this.GetCurrentCharacter()))
+            else if (char.IsWhiteSpace(this.GetCurrentCharacter()))
             {
-                this.TakeCharactersWhile(character => this.IsWhiteSpace(character));
+                this.TakeCharactersWhile(character => char.IsWhiteSpace(character));
```

Comment:
I'm not sure why I chose to use another implementation of IsWhiteSpace with only space, tab, new line, and carriage return back then. char.IsWhiteSpace matches more characters, as can be found in the [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.char.iswhitespace?view=netstandard-1.0#System_Char_IsWhiteSpace_System_Char_). Is it wise to change this?

Reply:
From my understanding it should be fine. It's not going to make a big change since those additional unicode white-spaces won't (ok, shouldn't) be used there. But `char.IsWhiteSpace` looks cleaner to me.